decred_wallet.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		25070A7F228DBD1500A36C17 /* Dcrlibwallet.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 25070A7D228DBD1500A36C17 /* Dcrlibwallet.framework */; };
```

Comment:
I'm not sure why the changes to this file are necessary.

Reply:
I did not make the changes. Xcode automatically did that when i added the file to the libs folder as instructed